ingest_data/sagemaker-keyspaces/SageMaker_Keyspaces_ml_example.ipynb

```diff
@@ -2,15 +2,16 @@
  "cells": [
```

Comment:
<p>Add a more explanatory title as to what this section does, in addition to "Optional"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
added more information, rearrange some cells